recipes/clair3/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "clair3" %}
+{% set version = "0.1.4" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: http://www.bio8.cs.hku.hk/clair3/bioconda/Clair3_v{{ version }}.zip
+  sha256: bc3a0aaf38f034a559b22ba7c8d29a16ebe62d810cf7903d6a7520ad93341106
+
+build:
+  number: 0
+  noarch: generic
+
+requirements:
+  run:
+    - python=3.6.10
+    - pypy3.6
```

Comment:
This requires pypy? Are the exact versions mentioned here really required, or are these instead minimum versions? The issue with extremely strict requirements is that this will quickly be unbuildable.

Reply:
Thanks for your comments, and yes, our tool requires pypy, and we only tested the whole workflow using pypy3.6.